### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by stefanruijsenaars**

### CODE REVIEW

File: src/Plugin/Block/LanguageSelectionPageBlock.php [hunk 1]
```diff
@@ -146,4 +148,25 @@ protected function config($name) {
     return $this->configFactory->get($name);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function blockAccess(AccountInterface $account) {
+    // @todo move this elsewhere, to the right place
+    $config = $this->config('language_selection_page.negotiation');
+    $manager = \Drupal::service('plugin.manager.language_selection_page_condition');
```
Comment (drupol) [inline src/Plugin/Block/LanguageSelectionPageBlock.php]: We should use proper dependency injection here.

**Comment by drupol** [comment]
Add:


```php
  /**
   * The Language Selection Page condition plugin manager.
   *
   * @var \Drupal\Core\Executable\ExecutableManagerInterface
   */
  protected $languageSelectionPageConditionManager;

  /**
   * PageController constructor.
   */
  public function __construct($configuration, $plugin_id, $plugin_definition, RequestStack $request_stack, ExecutableManagerInterface $plugin_manager) {
    parent::__construct($configuration, $plugin_id, $plugin_definition);
    $this->requestStack = $request_stack;
    $this->languageSelectionPageConditionManager = $plugin_manager;
  }

  /**
   * {@inheritdoc}
   */
  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
    return new static(
      $configuration,
      $plugin_id,
      $plugin_definition,
      $container->get('request_stack'),
      $container->get('plugin.manager.language_selection_page_condition')
    );
  }
```

File: src/Plugin/Block/LanguageSelectionPageBlock.php (hunk 1 above)
Comment (stefanruijsenaars) [inline src/Plugin/Block/LanguageSelectionPageBlock.php]: will do

Comment (stefanruijsenaars) [inline src/Plugin/Block/LanguageSelectionPageBlock.php]: updated

**Pull request merged**